src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -10,21 +11,17 @@ const SLIPPAGE_SETTINGS = {
 
 export function SwapSettingsSlippageInput({
   className,
-  defaultSlippage = 3,
 }: SwapSettingsSlippageInputReact) {
   const { updateLifecycleStatus, lifecycleStatus } = useSwapContext();
   const getMaxSlippage = useCallback(() => {
-    if (lifecycleStatus.statusName !== 'error') {
-      return lifecycleStatus.statusData.maxSlippage;
-    }
-    return defaultSlippage;
-  }, [lifecycleStatus.statusName, lifecycleStatus.statusData, defaultSlippage]);
+    return lifecycleStatus.statusData.maxSlippage;
+  }, [lifecycleStatus.statusData]);
```

Comment:
Side note, for next PR, I don't think we even need `getMaxSlippage` anymore. Right?

or am I missing something?

Reply:
Good point, we can remove it!

 I suppose the only benefit is minor performance improvements using memoization from `useCallback`